web/src/components/feature/MoleTable.tsx

```diff
@@ -0,0 +1,45 @@
+import { DynamicTable, TDynamicTableInput } from '../common/DynamicTable'
+import { ComponentResponse, MultiflashResponse } from '../../api/generated'
+
+function transformRawResponse(
+  multiFlashResponse: MultiflashResponse,
+  componentResponse: ComponentResponse
+): TDynamicTableInput {
+  const tableInput: TDynamicTableInput = { Components: [] }
```

Comment:
components in lowercase

Reply:
It's used here as the actual header of the table.
I'd rather not do any extra string manipulation that can take away flexibility of the header value.

Reply:
Could do `['Components']` to signal that it is meant to be a friendly string value. But eslint does not like that (Unnecessarily computed property ['Components'] found  no-useless-computed-key).